### DISCUSSION TITLE
Unpack vs Expand

### DISCUSSION THREAD

**Discussion by 93578237** [microsoft/pyright#3527] [Q&A] [answered]
Is Unpack[TypedDict] for annotating kwargs is a temporary workaround until Expand will be implemented in both mypy and pyright?
Why Unpack is allowing for **, when is for only * or am I missing something?

**Comment by JelleZijlstra** [comment, marked as answer]
`Unpack[]` was introduced by PEP 646 for `*args`. That is currently its only standardized use. An upcoming PEP, PEP 692, will allow using it for `**kwargs` too. Pyright already has experimental support for this feature.

`Expand` is a name that has been floating around in some proposals and in an unmerged mypy PR, but to my knowledge it has not been implemented in any type checkers. I do not expect it to become standardized.
